src/app/modules/forms/absence-form/absence-form.component.html

```diff
@@ -1,44 +1,68 @@
 <div class="absence-container">
-    <mat-card class="absence-form-wrapper">
-        <div class="absence-form-header">
-            <mat-card-title class="absence-form-title">Request Absence</mat-card-title>
-            <button mat-icon-button (click)="onClose()">
-                <mat-icon>close</mat-icon>
+  <mat-card class="absence-form-wrapper">
+    <div class="absence-form-header">
+      <mat-card-title class="absence-form-title">
+        {{ isEditMode ? 'Edit Absence' : 'Request Absence' }}
+      </mat-card-title>
+      <button mat-icon-button (click)="onClose()">
+        <mat-icon>close</mat-icon>
+      </button>
+    </div>
+    <mat-card-content>
+      <form [formGroup]="form" (ngSubmit)="onSubmit()" class="absence-form">
+        <mat-form-field appearance="fill">
+          <mat-label>What type of absence?</mat-label>
+          <mat-select formControlName="absenceType">
+            <mat-option value="sick">Sick</mat-option>
+            <mat-option value="vacation">Vacation</mat-option>
+          </mat-select>
+        </mat-form-field>
+        <div class="absence-date-form-container">
+          <mat-form-field appearance="fill" class="absence-date-box-container">
+            <mat-label>From</mat-label>
+            <input matInput
+                   [min]="currentDate.value.startOf('year').toISOString()"
+                   [max]="currentDate.value.endOf('year').toISOString()"
+                   [matDatepicker]="pickerFrom" formControlName="dateFrom">
+            <mat-datepicker-toggle matSuffix [for]="pickerFrom"></mat-datepicker-toggle>
+            <mat-datepicker #pickerFrom startView="month"></mat-datepicker>
+          </mat-form-field>
+          <mat-form-field appearance="fill" class="absence-date-box-container">
+            <mat-label>To</mat-label>
+            <input matInput
+                   [min]="currentDate.value.startOf('year').toISOString()"
+                   [max]="currentDate.value.endOf('year').toISOString()"
```

Comment:
please create pipe for this manipulations, it's a bad practice to call a function inside the template